### ISSUE TITLE
parameter tuning

### ISSUE THREAD

**Issue by mjcoo** [dmlc/xgboost#2923]
Hi, 
    I have a dataset for bioinfomatics. In real world data, the positive sample number: negative sample number = 1:60.  In my training data, the  positive sample number: negative sample number=1:1, and currently there are 400 features.  Actually I only care about the prediction accuracy and coverage for the positive class.

   How could I train my model to get good result in real world prediction.  Now I have grid searched the parameters: eta([0.001, 0.1,   0.5,  0.8]), gamma(0:2:22), max_depth(0:6:100) it seems the results will not change after max_depth is larger than 40. and the best performance(consider only the positive class) is Accuracy=TP/(TP+FP)=14%, Coverage=TP/(TP+FN)=3.2%.

Then I also tuned min_child_weight(1:1:5)max_delta_step(0:1:5)max_delta_step(0:1:5),scale_pos_weight(0:1:5),
but the results are still unsatisfactory.

Any suggestions for my situation?
